### DISCUSSION TITLE
`Error Unauthorized action or expired link`

### DISCUSSION THREAD

**Discussion by dzg** [YOURLS/YOURLS#3774] [Q&A] [answered]
When logging in/out as various users, I keep getting this `Error Unauthorized action or expired link` – but only sometimes.

It comes from the url `https://mysite.com/admin/` – but if I then go to `https://mysite.com/` without the `admin` then I am logged in.

**Comment by dzg** [comment]
Sometimes it seems to redirect to `https://xrgo.run/admin/index.php?` which also gives the error

**Comment by dzg** [comment]
Might be my`.htaccess` ...

**Comment by dgw** [comment]
YOURLS uses nonces to protect admin actions, and the error string you included comes from the nonce verification function. Since you say "as various users", I would guess you end up with multiple tabs open sometimes.

YOURLS nonces expire after some time, and are also associated with specific users. For example, a tab you opened while logged in as user1 can contain old nonces that will no longer work if you have logged out and then back in as user2 in a different browser tab.

**Comment by YouveGotMeowxy** [comment]
@dzg Did you ever figure this out? I'm having it happen to me, and after reading @dgw comment I'm still confused as to why it's happening.

I don't have multiple users; just my 1 admin account, and ever single time I login it happens. Even if I login, immediately logout, then login again.

> Error
> Unauthorized action or expired link

The page is:

> https://mysite.com/x/admin/index.php

Then I have to manually remove `index.php` so it's like this `https://mysite.com/x/admin/` to go to the regular admin page.

Does anyone know where I need to go (file config, setting, plugin, etc.) to get this to not happen and just show my usual admin page?

**Reply by dgw** [reply, marked as answer]
If you have installed Sleeky… that's probably the cause. https://github.com/YOURLS/YOURLS/issues/3087#issuecomment-2013220165

**Reply by YouveGotMeowxy** [reply]
@dgw you're good! I do have it installed, and the fix worked! TY!!
